### ISSUE TITLE
Automatic Update of the tw.org website when we release

### ISSUE THREAD

**Issue by felixschurk** [GothenburgBitFactory/taskwarrior#3631]
Pointed out in #3630, it seems like the website is not automatically updated when a new version of taskwarrior is released.

**Comment by djmitche** [comment]
By automation, are you thinking a GH action to create a PR on tw.org?

**Comment by felixschurk** [comment]
Yes, I assume something in this direction would make sense. However I never had a chance to look into this kind of connections so I am not aware on how easy one could like that.

**Comment by ryneeverett** [comment]
While the "push strategy" of opening a PR on release might be better, the "pull strategy" of polling the taskwarrior repo for a higher version tag from the two.org repo with a cron job like the "update-tools" one they're already using might be easier to implement. Likewise, it also might be easier to use the "merge-pull-requests-action" to automatically merge such pull requests originating from within the tw.org repo than those originating from a different repo in the organization. Easier isn't always better, of course.
